tests/jobs/misfit_preprocessor/test_misfit_preprocessor.py

```diff
@@ -72,8 +73,8 @@ def assert_homogen_clusters(config):
 
 
 def generate_simulated_responses(
-    forward_polynomials,
-    parameter_distribution,
+    _forward_polynomials,
+    _parameter_distribution,
```

Comment:
Are these two parameters even used? I tried to remove them completely and I don't think it broke anything ...

Reply:
They are not used in this test no, but I guessed it was nice to keep them in for illustration. The underscore just silences `pylint`.

Reply:
The test is changed such that it uses forward polynomials and parameter_distribution